tests/test_table.py

```diff
@@ -1118,22 +1122,25 @@ def test_load(tmpdir):
     with pytest.raises(EOFError):
         table_loaded.load(path_no_ext)
 
-    # repeat with CSV file as fall back
-    table.save(
-        path_no_ext,
-        storage_format=audformat.define.TableStorageFormat.CSV,
-    )
-    with open(path_pkl, "wb"):
-        pass
-    table_loaded = audformat.Table()
-    table_loaded.columns = table.columns
-    table_loaded._db = table._db
-    table_loaded.load(path_no_ext)
-    pd.testing.assert_frame_equal(table.df, table_loaded.df)
+    # repeat with CSV|PARQUET file as fall back
+    for ext in [
+        audformat.define.TableStorageFormat.CSV,
+        audformat.define.TableStorageFormat.PARQUET,
+    ]:
+        table.save(path_no_ext, storage_format=ext)
+        with open(path_pkl, "wb"):
+            pass
+        table_loaded = audformat.Table()
+        table_loaded.columns = table.columns
+        table_loaded._db = table._db
+        table_loaded.load(path_no_ext)
+        pd.testing.assert_frame_equal(table.df, table_loaded.df)
 
-    # check if pickle file was recovered from CSV
-    df = pd.read_pickle(path_pkl)
-    pd.testing.assert_frame_equal(table.df, df)
+        # check if pickle file was recovered
+        df = pd.read_pickle(path_pkl)
+        pd.testing.assert_frame_equal(table.df, df)
```

Comment:
I remember that I had some strange asymmetry concerning the results of `pd.testing.assert_frame_equal`  and `pandas.DataFrame.equals`. Digging into the code seems to also suggests that the implementation in the testing module does not rely on the core data frame function.

Reply:
`pandas.DataFrame.equals` does not check for the same dtype:

![image](https://github.com/audeering/audformat/assets/173624/c450ad37-33d8-4724-8212-a4ff7b378d68)

`pandas.testing.assert_frame_equal` does:

![image](https://github.com/audeering/audformat/assets/173624/3d9c2caf-ac87-4dc0-9a40-79bb407012bc)

So, if you really want to ensure that two dataframes are identical in a test, `pandas.testing.assert_frame_equal` should be the choice.

Reply:
So testing the typing is an addition that comes with asserts from `pd.testing`.
Then I will resolve as this is the stricter test, sth like `assert df1.equals(df2)` is not necessary.